### ISSUE TITLE
Modularity (Q value) for each cluster

### ISSUE THREAD

**Issue by sgautam666** [vtraag/leidenalg#169]
It seems like the current partition method can return modularity for the cluster result. Is it possible to get the Q-value to each individual clusters?

**Comment by vtraag** [comment]
No, no such values per community are returned. But you can easily calculate this yourself using standard python and functionality of `igraph`. If you are not able to figure it out, let me know.

**Comment by sgautam666** [comment]
It would be nice if you have an example already that you can share. Thank you
